src/Data/Textile/Query.elm

```diff
@@ -448,12 +458,12 @@ default =
     , makingWaste = Nothing
     , mass = Mass.kilograms 0.17
     , materials =
-        [ { country = Nothing
-          , id = Material.Id "ei-coton"
-          , share = Split.full
-          , spinning = Nothing
-          }
-        ]
+        case Material.idFromString "62a4d6fb-3276-4ba5-93a3-889ecd3bff84" of
+            Err _ ->
+                []
```

Comment:
Please avoid silencing errors. This function should return a `Result` instead of an empty query, with an error when the uuid isn't found (which may happen at some point if we remove the matching material)